index.js

```diff
@@ -46,6 +47,7 @@ function createFile (name, opts) {
   }
 
   function close (req) {
+    toDestroy = entry
```

Comment:
i think you want to set this https://github.com/random-access-storage/random-access-chrome-file/blob/b746d8651b8c7c30625e53037cf56ef1faa09753/index.js#L80 instead

Reply:
`entry = toDestroy = e`?